spring-cloud-function-gateway/src/main/java/org/springframework/cloud/function/gateway/ProxyExchange.java

```diff
@@ -0,0 +1,611 @@
+/*
+ * Copyright 2012-2015 the original author or authors.
```

Comment:
Spring Boot has a checkstyle rule that rejects "2017" but I believe you are rigth.